src/packages/recompose/toClass.js

```diff
@@ -7,16 +7,17 @@ const toClass = baseComponent => {
     return baseComponent
   }
 
-  return class extends Component {
-    static displayName = getDisplayName(baseComponent)
-    static propTypes = baseComponent.propTypes
-    static contextTypes = baseComponent.contextTypes
-    static defaultProps = baseComponent.defaultProps
-
+  const NewComponent = class extends Component {
```

Comment:
The convention used in the other modules is for the class name to be the uppercase version of the helper. Also, use a normal class, not a class expression. So this would be:

```js
class ToClass extends Component {...}
```